libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -723,6 +724,8 @@ IS: 'IS';
 NULL: 'NULL';
 TRUE: 'TRUE';
 FALSE: 'FALSE';
+IGNORE: 'IGNORE';
+RESPECT: 'RESPECT';
```

Comment:
We'll have to add these two to the `nonReserved` list - otherwise this would be a breaking change as users would no longer be able to use these identifiers without quotes

Reply:
sorry :( tried to get it working first then forgot to fix it